R/chartutils.R

```diff
@@ -1182,6 +1182,9 @@ setValRange <- function(min, max, values, show.zero = FALSE, use.defaults = TRUE
         min <- charToNumeric(min)
         max <- charToNumeric(max)
     }
+    # When there is only a single value don't use it to determine range
+    if (length(values) == 1)
```

Comment:
What if there are multiple values with the same value? Would there be the same issue as a single value?

Reply:
Yep, changed the condition to `length(unique(values))` in [a52e9a1](https://github.com/Displayr/flipStandardCharts/pull/117/commits/a52e9a1562d93699dd331f6ce6ba6311df3745f4)